src/main/java/nextstep/app/ui/BasicAuthenticationInterceptor.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.app.ui;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import nextstep.app.util.Base64Convertor;
+import nextstep.security.UserDetails;
+import nextstep.security.UserDetailsService;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+public class BasicAuthenticationInterceptor implements HandlerInterceptor {
+    private final UserDetailsService userDetailsService;
+
+    public BasicAuthenticationInterceptor(UserDetailsService userDetailsService) {
+        this.userDetailsService = userDetailsService;
+    }
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        try {
+            String authorization = request.getHeader("Authorization");
```

Comment:
HttpHeaders.AUTHORIZATION 를 사용해봐도 좋을 것 같아요 😄
```suggestion
            String authorization = request.getHeader(HttpHeaders.AUTHORIZATION);
```

Reply:
앗 그렇네요ㅠㅠ
그런데 Interceptor 는 더이상 사용하지 않아서 삭제하겠습니다!